docs/cri/config.md

```diff
@@ -32,6 +32,18 @@ cgroupDriver: "systemd"
 
 kubeadm users should also see [the kubeadm documentation](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).
 
+> Note: Kubernetes v1.28 supports automatic detection of the cgroup driver as
+> an alpha feature. With the `KubeletCgroupDriverFromCRI` kubelet feature gate
+> enabled, the kubelet automatically detects the cgroup driver from the CRI
+> runtime and the `KubeletConfiguration` configuration step above is not needed.gq
+>
+> When determining the cgroup driver, containerd uses the `SystemdCgroup`
+> setting from runc-based runtime classes, starting from the default runtime
+> class. If no runc-based runtime classes have been configured containerd
+> relies on auto-detection based on determining if systemd is running.
```

Comment:
```suggestion
> relies on auto-detection based on determining if systemd is running.
> containerd v1.x always defaults to non-systemd cgroupfs driver.
```

Reply:
Hmm, I'm not sure about this 🧐 What I mean is that this PR only adds new functionality that instructs the kubelet which cgroup driver it should use (for pod sandbox-level cgroups). Containerd v1.x doesn't have this functionality. The PR doesn't change any internal configuration logic or defaults of containerd itself. There is no "containerd-wide" (or even "containerd.cri -wide") cgroup driver setting. The setting only exists in the runc runtime handler plugin options. For other runtime handlers containerd doesn't care (afaiu).

Reply:
containerd v1.x doesn't have this functionality .. "yet". 1.6 is LTS and may need to support the api before EOL